docs/develop/json-manifest-overview.md

```diff
@@ -0,0 +1,433 @@
+---
+title: Teams manifest for Office Add-ins (preview)
+description: Get an overview of the preview JSON manifest.
+ms.date: 05/24/2022
+ms.localizationpriority: high
+---
+
+# Teams manifest for Office Add-ins (preview)
+
+Microsoft is making a number of improvements to the Microsoft 365 developer platform. These improvements provide more consistency in the development, deployment, installation, and administration of all types of extensions of Microsoft 365, including Office Add-ins. These changes are compatible with existing add-ins. 
+
+Two important features we are aiming at, *but are not yet available*:
+
+- It should be possible to surface a single web app as multiple types of Microsoft 365 extensions. For example, a web app can be both an Office Add-in and a custom tab in Teams.
+- All types of Microsoft 365 extensions should use the same manifest format and schema. It will be based on the current JSON-formatted Teams manifest schema. 
+
+We have taken an important first step toward these goals by making it possible for Outlook Add-ins, running on Windows only, to be created with a version of the Teams JSON manifest.
+
+> [!NOTE]
+> The new manifest is available for preview and is subject to change based on feedback. We encourage experienced add-in developers to experiment with it. The preview manifest should not be used in production add-ins. 
+
+During the early preview period, the following limitations apply.
+
+- The preview version of the Teams manifest only supports Outlook add-ins and only on subscription Office for Windows. We're working on extending support to Excel, PowerPoint, and Word.
+- It is not yet possible to combine an add-in with Teams apps, such as a custom Teams tab, or any other Microsoft 365 extension types. So, you cannot yet add an add-in to an existing Teams app. This will be available before release to general availability.
```

Comment:
"It is not yet possible to combine and sideload an add-in with Teams apps, such as a Teams personal tab, or other Microsoft 365 extension types.  In the coming months, we will continue to extend the preview to support these scenarios and provide additional tools to update manifests to the preview format."